AxonDeepSeg/segment.py

```diff
@@ -108,10 +139,38 @@ def segment_image(
 
         img_name_original = acquisition_name.stem
 
+        # Get the model's native resolution
+        model_resolution, patch_size = get_model_native_resolution_and_patch(path_model)
+
+        # Check that the resampled image will be of sufficient size, and if not throw an error.
+        im = Image.open(path_testing_image)
+        w, h = im.size
+
+        w_resampled = w*(acquired_resolution*zoom_factor)/model_resolution
+        h_resampled = h*(acquired_resolution*zoom_factor)/model_resolution
+
+        if w_resampled < patch_size or h_resampled < patch_size:
```

Comment:
Same as above for "<" with list for patch_size
```
TypeError: '<' not supported between instances of 'float' and 'list'
```

Reply:
Same fix as above, but this time using min instead of max:
```python
if patch_size[0] == patch_size[1]: #isotropic pixels
   patch_size = patch_size[0]
else:
   patch_size = min(patch_size)
```